client/src/App.js

```diff
@@ -8,38 +8,35 @@ import Board from './components/pages/Board';
 import Alert from './components/other/Alert';
 
 // Redux
-import { Provider } from 'react-redux';
-import store from './store';
-import { loadUser } from './actions/auth';
 import setAuthToken from './utils/setAuthToken';
 
 import './App.css';
+import {AuthContext} from "./contexts/authStore";
+import BoardStore from "./contexts/boardStore";
```

Comment:
why are these two being imported differently? Uniformity is very important.

Reply:
Both are different. BoardStore is the store wrapper and AuthContext is to be used with useContext

Reply:
the AuthStore (like BoardStore here) is imported is used in index.js